src/ci/resources/gcp_batch_application.conf

```diff
@@ -65,5 +65,11 @@ backend {
         include "gcp_batch_provider_config.inc.conf"
       }
     }
+    GCPBATCHGoogleSecretManager {
+      actor-factory = "cromwell.backend.google.batch.GcpBatchBackendLifecycleActorFactory"
+      config {
+        include "gcp_batch_provider_config.inc.conf"
```

Comment:
Nice, looks like since this is configured per backend, it won't be a problem to have the PAPI and Batch backends authing with different tokens while they're both in use.